10XGenomics/enclone
# various changes

VARIABLES
- There is a new page that inventories all the enclone variables.
- Almost any parseable variable can now be used in a linear condition. 
- A number of other restrictions on variable use were removed.

PROFILE
- Turn on overflow checking in release mode because we might detect new customer problems.
- Set debug = 1 in test mode for consistency.
- Add comments on some experiments.

DATA IMPORT
- Handle the case where the directory already exists.
- Find analysis directory in per_sample_outs.

TESTING
- Document how to run just one test on the developer's page.
- Make it easier to add a single test to the TESTS list in testlist.rs; now one just types run_last_test.
- Add internal EVIL_EYE option to help trace weird hangs.
- Revise link test to be aware of links that will exist after page goes live.

OTHER
- Update crates.
- Tweak font advice seen if one types "enclone help".
- Support alternative inequality symbols.
- Clicking on the enclone banner at the top of any page on the site now takes you back to the help section on main page.
- Slightly relax unaccounted time check.
- Accept 0.2% decrease in CPU.
- Test if customer supplies the wrong kind of reference with REF=....
